### PR TITLE
add check to CI that Bitcoin only firmware does not contain altcoin code

### PR DESCRIPTION
**Pull request by prusnak**
Last bits of the Bitcoin only firmware

### CODE REVIEW

**Comment by matejcik** [comment]
...of course, it's failing the gen check. the seds would have to be in `do_build`, and pass an argument whether to use them

**Comment by prusnak** [comment]
54fbb4f is ready for review and merge!

**Comment by prusnak** [comment]
a66236f4b2ee4fa058794d3f6f8603d97191aabe is ready for review and merge!

**Comment by tsusanka** [comment]
LGTM. Fast untested ACK.

File: core/tests/test_apps.monero.serializer.py [hunk 1]
```diff
@@ -2,72 +2,75 @@
...
+    from apps.monero.xmr.serialize_messages.tx_prefix import (
+        TxinGen,
+        TxinToKey,
+        TxInV,
+        TxOut,
+        TxoutToKey,
+    )
+
+
+if not utils.BITCOIN_ONLY:
```
Comment (matejcik) [inline core/tests/test_apps.monero.serializer.py]: this could be in the same block as the above, no?

Comment (prusnak) [reply core/tests/test_apps.monero.serializer.py]: I'd rather keep this explicit (i.e. conditional imports and conditionally defined classes separated)

**Comment by prusnak** [comment]
Merged via 2fe34aa33b1a208314592c0208d1f082f5ad4a77

**Pull request closed without merging**